lane.py

```diff
@@ -1645,335 +1476,32 @@ def works_for_hits(self, _db, hits, work_model=mw):
 
         b = time.time()
         logging.info(
-            u"Obtained %sx%d in %.2fsec", work_model.__name__, len(results), b-a
+            u"Obtained %sxWork in %.2fsec", len(results), b-a
         )
         return results
 
-    def apply_filters(self, _db, qu, facets, pagination, featured=False):
```

Comment:
All this code was removed from `WorkList` and quarantined in the new `DatabaseBackedWorkList` class.